hieradata/common.yaml

```diff
@@ -1117,6 +1117,10 @@ hosts::production::management::hosts:
     ip: '10.1.0.3'
     legacy_aliases:
       - "deploy.%{hiera('app_domain')}"
+  deploy-1:
+    ip: '10.1.0.4'
+    legacy_aliases:
+      - "deploy.%{hiera('app_domain')}"
```

Comment:
I think this will conflict, can we change it to something different during testing?